src/Illuminate/Foundation/Console/KeyGenerateCommand.php

```diff
@@ -35,10 +35,16 @@ class KeyGenerateCommand extends Command
      */
     public function handle()
     {
+        if ($this->shouldSaveExistingKey()) {
```

Comment:
I feel like this could be a breaking change for someone currently using this command as part of a deployment, probably should target 12.x